server/worker/zendesk.go

```diff
@@ -222,9 +226,6 @@ func (z *Zendesk) Name() string {
 
 // zendeskArgs are the arguments for the Zendesk integration job.
 type zendeskArgs struct {
-	// CVE is deprecated but kept for backwards compatibility (there may be jobs
```

Comment:
@mna @gillespi314 I removed this to do a little bit of clean up, but this was done under the assumptions that:
1. Since the new payload format was introduced a long time ago (~6 months), all jobs using the 'old' format would be finish by now.
2. Users are not able to retry jobs that succeeded.

Am I correct?

Reply:
@juan-fdz-hawa I think this is risky, as some deployments may still be on the old version that used the old job format (I don't remember for how long this version was deployed, but even if it was just for one release, we may have users on that release at the moment), and that will upgrade directly to the new version.

Reply:
Thanks @mna I'll revert this back

Reply:
Yeah I think it's best, we've seen users migrate from pretty old versions / skip a lot of versions. I think it's the kind of backwards compatible code we may need to keep until a new major version.